src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -31,6 +31,9 @@ public abstract partial class ViewHandler : ElementHandler, IViewHandler
 			new PropertyMapper<IView, IViewHandler>(ElementHandler.ElementMapper)
 #endif
 			{
+				// This property is a special one and needs to be set before other properties.
+				[nameof(IViewHandler.ContainerView)] = MapContainerView,
```

Comment:
There are some ContainerView related tests failing on iOS and Catalyst:
![image](https://github.com/user-attachments/assets/c47d6346-f80d-4ca4-a276-040056a15f81)

For example:

ContainerViewInitializesCorrectly
Assert.Equal() Failure: Values differ\nExpected: True\nActual:   False

Could you review it? Let me know if need help with something.

Reply:
I'm convinced my change is correct and I just revealed a race condition issue.
I pushed a fix to the real underlying issue in `ContainerViewController`.
Can you run AZP again? Thanks